pyEpiabm/pyEpiabm/sweep/initial_demographics_sweep.py

```diff
@@ -0,0 +1,90 @@
+#
+# Sweep to record demographic information of the population in a .csv file
+#
+
+import os
+import typing
+
+from pyEpiabm.core import Parameters
+from pyEpiabm.output import _CsvDictWriter
+
+from .abstract_sweep import AbstractSweep
+
+
+class InitialDemographicsSweep(AbstractSweep):
+    """Class to sweep through the population at the beginning of the simulation
+    and record their demographic information
```

Comment:
```suggestion
    and record their demographic information.
```

Reply:
Ditto for other function docstrings - compare to existing style

Reply:
Will do